src/components/Pages/Student/StudentGeogebra.vue

```diff
@@ -22,11 +19,11 @@
                         </router-link>
                     </div>
                 </div>
-                <div class="ibox-content">
-                    <div class="geogebra-view">
+                <div class="ibox-content geogebra-frame">
+                    <div id="geogebra-view">
                         <!--Geogebra applet-->
                     </div>
-                    <div class="control-options">
+                    <!-- <div class="control-options">
```

Comment:
Why do we have stuff commented out?

Reply:
Because the layout represents part of the UI related to controls. I agreed with Chris to leave it for now, but I dont want to delete the layout.